core/src/main/scala-3/spire/syntax/macros/cforMacros.scala

```diff
@@ -94,54 +96,54 @@ def fastForRangeMacro(r: Expr[Range], body: Expr[Int => Unit])(using quotes: Quo
   import quotes.reflect._
 
   def strideUpUntil(fromExpr: Expr[Int], untilExpr: Expr[Int], stride: Expr[Int]): Expr[Unit] = '{
-      var index = $fromExpr
-      val limit = $untilExpr
-      while (index < limit) {
-        ${ Expr.betaReduce(body) }(index)
-        index += $stride
-      }
+    var index = $fromExpr
+    val limit = $untilExpr
+    while (index < limit) {
+      ${ Expr.betaReduce(body) }(index)
+      index += $stride
     }
+  }
 
   def strideUpTo(fromExpr: Expr[Int], untilExpr: Expr[Int], stride: Expr[Int]): Expr[Unit] = '{
     var index = $fromExpr
-    val end   = $untilExpr
+    val end = $untilExpr
     while (index <= end)
-      ${ Expr.betaReduce(body) }(index)
-      index += $stride
```

Comment:
hmm??

Reply:
I would expect `scalafmt` not to change the semantics of the code

Reply:
Yeah seems like a bug. 😠